conf/app.routes

```diff
@@ -10,5 +10,7 @@ HEAD       /traders/:eori/checkRecords              uk.gov.hmrc.tradergoodsprofi
 HEAD       /traders/:eori/records/storeLatest       uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.StoreRecordsController.storeLatestRecords(eori: String)
 HEAD       /traders/:eori/records/store             uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.StoreRecordsController.storeAllRecords(eori: String)
 GET        /traders/:eori/records/count             uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.GetRecordsController.getRecordsCount(eori: String)
+GET        /traders/:eori/records/searchCount       uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.SearchRecordsController.getRecordsCount(eori: String, searchString: String)
```

Comment:
why do we have an endpoint for the searchCount?

Reply:
You are right. This is not needed

Reply:
Updated